frontend/src/lib/utils/responsiveUtils.ts

```diff
@@ -16,12 +25,15 @@ export function parsePixelValue(cssStatement: string): number {
 
 export function getActiveBreakpoint(): number {
     const { innerWidth: width } = window
-    const breakpoints = Object.values(responsiveMap).map((cssStatement) => parsePixelValue(cssStatement))
-    let breakpoint = breakpoints[0]
-    breakpoints.forEach((value) => {
+    let breakpoint = BREAKPOINT_VALUES[0]
+    BREAKPOINT_VALUES.forEach((value) => {
         if (width > breakpoint) {
             breakpoint = value
         }
     })
     return breakpoint
 }
```

Comment:
I'd also simplify this to:

```js
export function getActiveBreakpoint(): number {
    const width = window.innerWidth
    const lastMatchingBreakpoint = BREAKPOINT_VALUES.filter(value => width >= value).pop()
    return lastMatchingBreakpoint || BREAKPOINT_VALUES[0]
}
```

Reply:
I also renamed this function `getActiveBreakpointValue` (over `getActiveBreakpoint`) since technically a "breakpoint" is one of "sm", "xs", etc, not the value in pixels it resolves to.

And I changed `width > value` to `width >= value`, as that's how (antd) breakpoints [are defined](https://ant.design/components/grid/#Col).